packages/client/src/utils/referenceApi.ts

```diff
@@ -74,6 +79,10 @@ export interface IApplicationConfig {
   PHONE_NUMBER_PATTERN: RegExp
   NID_NUMBER_PATTERN: RegExp
   ADDRESSES: number
+  INTEGRATIONS?: [IIntegration]
```

Comment:
Why is it that INTEGRATIONS was added here? They are actually added in quite a few places in this PR. We should remove them